tests/integration/test_matrix.py

```diff
@@ -1457,3 +1457,54 @@ def it_computes_mr_x_mr_table_stderr(self):
                 ]
             ),
         )
+
+    @pytest.mark.parametrize(
```

Comment:
I don't see that this has an `xfail` at this particular commit (`d4981c133f75edd514cedc873656353cb3680daa`).